source/css/sass/mixins/_typography.scss

```diff
@@ -109,3 +109,12 @@
 @mixin label-tag-typography() {
   @include _label-typography($color-primary-normal);
 }
+
+@mixin hidden-link($hover-color: $color-primary-dark) {
```

Comment:
Going to extract this in a separate PR.

Reply:
> Going to extract this in a separate PR.

Should I remove any reference to it in this PR?